modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/DefinitionInfo/DetailsTab.tsx

```diff
@@ -37,30 +39,36 @@ export function DetailsTab({definitionInfo}: DetailsTabProps) {
 			: `${totalModifications} ${Liferay.Language.get('revision')}`;
 
 	return (
-		<>
-			<div className="info-group">
-				<label className="text-secondary">
+		<div className="lfr-workflow__details-tab-container">
+			<div className="lfr-workflow__details-tab-info-container">
+				<label className="lfr-workflow__details-tab-info-title">
 					{titleCreated.toUpperCase()}
 				</label>
 
-				<span>{dateCreated}</span>
+				<span className="lfr-workflow__details-tab-info-details">
+					{dateCreated}
+				</span>
 			</div>
 
-			<div className="info-group">
-				<label className="text-secondary">
+			<div className="lfr-workflow__details-tab-info-container">
+				<label className="lfr-workflow__details-tab-info-title">
 					{titleLastModified.toUpperCase()}
 				</label>
 
-				<span>{dateModified}</span>
+				<span className="lfr-workflow__details-tab-info-details">
+					{dateModified}
+				</span>
 			</div>
 
-			<div className="info-group">
-				<label className="text-secondary">
+			<div className="lfr-workflow__details-tab-info-container">
+				<label className="lfr-workflow__details-tab-info-title">
 					{titleTotalModifications.toUpperCase()}
 				</label>
 
-				<span>{revisionMessage}</span>
+				<span className="lfr-workflow__details-tab-info-details">
+					{revisionMessage}
+				</span>
 			</div>
-		</>
+		</div>
```

Comment:
I see that we can improve this component, this code snippet is repeated tree times:

```
	<div className="lfr-workflow__details-tab-info-container">
		<label className="lfr-workflow__details-tab-info-title">
			{x}
		</label>

		<span className="lfr-workflow__details-tab-info-details">
			{y}
		</span>
	</div>
```

We can create an array to store the constants declared above and loop through the array to render these contents